testsuite/MDAnalysisTests/parallelism/test_multiprocessing.py

```diff
@@ -125,6 +141,21 @@ def test_universe_unpickle_in_new_process():
     assert_equal(ref, res)
 
 
+def test_creating_multiple_universe_without_offset(temp_xtc, ncopies=3):
+    #  test if they can be created without generating
+    #  the offset simultaneously.
+    #  The tested XTC file is way too short to induce a race scenario but the
+    #  test is included as documentation for the scenario that used to create
+    #  a problem (see PR #3375 and issues #3230, #1988)
+
+    args = (GRO, str(temp_xtc))
+    with multiprocessing.Pool(2) as p:
+        universes = [p.apply(mda.Universe, args) for i in range(ncopies)]
```

Comment:
@yuxuanzhuang why are we using [`apply`](https://docs.python.org/3/library/multiprocessing.html#multiprocessing.pool.Pool.apply), which blocks, and not `apply_async`?

Reply:
ping @yuxuanzhuang

Reply:
Okay, I may have misunderstood `apply` as opposed to `apply_async`. I have changed accordingly.

Reply:
I think `apply` never runs the tasks in parallel (I don't even know why it exists right now:<) It is just a wrapper of `apply_async().get()` [under the hood](https://github.com/python/cpython/blob/main/Lib/multiprocessing/pool.py#L352).

```python
import multiprocessing
import os

def f():
    time.sleep(1)
    return 0

n_works = 4

with multiprocessing.Pool(n_works) as pool:
    multiple_workers = [pool.apply_async(f, ()) for i in range(n_works)]
    print([res.get() for res in multiple_workers])
1.7 s

with multiprocessing.Pool(n_works) as pool:
    print([pool.apply(f, ()) for i in range(n_works)])
4.1 s
```
So the answer is `apply_async` is definitely what we want to test here. It also doesn't trigger the test when the trajectory is short.

Reply:
Thank you for doing the research. I am very happy with the change then.